pyogrio/tests/test_geopandas_io.py

```diff
@@ -75,6 +75,27 @@ def test_read_dataframe_vsi(naturalearth_lowres_vsi):
     assert len(df) == 177
 
 
+@pytest.mark.parametrize("use_arrow", [True, False])
+@pytest.mark.parametrize(
+    "columns, fid_as_index, exp_len", [(None, False, 2), ([], True, 2), ([], False, 0)]
+)
+def test_read_layer_without_geometry(
+    test_fgdb_vsi, columns, fid_as_index, use_arrow, exp_len
+):
+    if use_arrow and (not has_pyarrow or __gdal_version__ < (3, 6, 0)):
```

Comment:
this can move up to the parametrize statement:

```
@pytest.mark.parametrize("use_arrow", [
    pytest.param(True, marks=pytest.mark.skipif(not has_pyarrow or __gdal_version__ < (3, 6, 0), reason="Arrow tests require pyarrow and GDAL>=3.6")), 
    False
])
```

Reply:
Yes, I saw this construction in other tests, but I don't think it is very readable... 

But, as you seem to have a preference for it I'll change it.